### ISSUE TITLE
Watermark removal

### ISSUE THREAD

**Issue by azsunlight** [ExponentialML/Text-To-Video-Finetuning#28]
I fine-tuned the model via single video fine tuning, but the watermark is still there. Would like to know fine-tune detail that can remove the watermark

Many thanks

**Comment by azsunlight** [comment]
And also what dataset do you use for the fine-tuning. thx

**Comment by ExponentialML** [comment]
Hey @azsunlight. 

Usually training the temporal attention layers `('attn1', 'attn2') ` at a learning rate of roughly `1e-5` and weight_decaty `0` should be enough to get rid of the watermark.

**Comment by HuaizheXu** [comment]
Hi, thanks for the information. Does the single-video dataset work as well?

**Comment by azsunlight** [comment]
Hi @ExponentialML, thanks for the information. Does single-video fine-tuning work as well?

**Comment by green-s** [comment]
@ExponentialML Could [this](https://github.com/rohitgandikota/erasing) technique potentially be applied to remove the watermark?

**Comment by ExponentialML** [comment]
@green-s & @azsunlight There was a [PR](https://github.com/ExponentialML/Text-To-Video-Finetuning/pull/39) that got merged with this feature in the inference script. Does this resolve the issue for you two?

**Comment by azsunlight** [comment]
> @green-s & @azsunlight There was a [PR](https://github.com/ExponentialML/Text-To-Video-Finetuning/pull/39) that got merged with this feature in the inference script. Does this resolve the issue for you two?

Yes. It works. Thank you
